lib/web_ui/dev/build.dart

```diff
@@ -114,6 +128,8 @@ class GnPipelineStep extends ProcessStep {
 
   String get runtimeModeFlag {
     switch (runtimeMode) {
```

Comment:
This could/should just be `runitemMode.name` I think - https://api.dart.dev/stable/2.19.6/dart-core/EnumName/name.html

Reply:
Yep, good call.